classes/Carrier.php

```diff
@@ -898,6 +904,7 @@ public function getGroups()
      * Clean delivery prices (weight/price).
      *
      * @param string $rangeTable Table name to clean (weight or price according to shipping method)
+     * @param mixed $range_table
```

Comment:
either change parameter to `$rangeTable` or change the PHPDoc above with `range_table`

Reply:
and `string` type of course :)